### PR TITLE
Driver commissions with invalid customers

### LINKED ISSUE #61: Driver commission histories don't return records when join to funny customer ids
Some driver commission history records have funny values in the `CustomerId` column that don't correspond to the same customer as seen in Grossman.

Some silly buggers are going on here, but since the customer details are not needed for driver commissions we can just ignore them for now.

### PR DESCRIPTION
**Pull request by josephbridgwaterrowe**
Update the driver commission history model to just include the customer
so it is a left join which will handle funny (non customer id) data
in the CustomerId of the Trucking_Drivers_Commissions_History schema.

This could be specific to our (Western Milling) Grossman/Relativity
schemas but for our use this isn't a problem for now.

Closes #61

### CODE REVIEW

**Comment by josephbridgwaterrowe** [comment]
TLDR;

For some reason there were driver commission `CustomerId` values with `31    00` which don't relate to any `CustomerId` in the `Customers` "table".

Honestly I don't have the will to find out what is going on, but I suspect its either due to a schema change/mismatch with the underlying data (some transactions are putting the data in different places within the COBOL data file).

This changes the `Customers` `INNER JOIN` to ` LEFT JOIN` so that those commission records are still returned.

This isn't really a problem at present since the driver commissions payroll export doesn't need to do anything with the customer details... maybe further down the road we may have to revisit this.
